### PR TITLE
Add Hack Camp 2016 blog post

### PR DESCRIPTION
**Pull request by MaxWofford**
Not yet finished

### CODE REVIEW

**Comment by zachlatta** [comment]
Let's make this a bit longer with more details (like testimonials) before
posting.

On Thu, Apr 28, 2016 at 2:31 PM, Max Wofford <notifications@github.com>
wrote:

> Not yet finished
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/hackclub/blog/pull/25
> Commit Summary
>
>    - Add Hack Camp 2016 blog post
>
> File Changes
>
>    - *A* _posts/2016-04-28-hack-camp-is-back.md
>    <https://github.com/hackclub/blog/pull/25/files#diff-0> (11)
>
> Patch Links:
>
>    - https://github.com/hackclub/blog/pull/25.patch
>    - https://github.com/hackclub/blog/pull/25.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/blog/pull/25>
>



-- 
Zach Latta
Hack Club | zach@hackclub.com

File: _posts/2016-04-28-hack-camp-is-back.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout:     post
+title:      Hack Camp Is Back
+date:       2016-04-28
```
Comment (zachlatta) [inline _posts/2016-04-28-hack-camp-is-back.md]: Update to today.

File: _posts/2016-04-28-hack-camp-is-back.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+author:     Max Wofford
+summary:    We'll be running a Hack Camp this summer.
```
Comment (zachlatta) [inline _posts/2016-04-28-hack-camp-is-back.md]: Change to "Hack Camp is returning this summer"!

File: _posts/2016-04-28-hack-camp-is-back.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+> feature I had previously thought impossible to implement. It really taught me
+> that I could 'distort reality' and make things HAPPEN.
+
+> The way Hack Camp is taught is totally opposite of what our school system is.
+> This camp really opened my mind to the different ways that I can learn.
+
+>  In the camp's mere two weeks, more than in any part of my life, not in my
+>  days as a fetus, an infant, a toddler, an elementary schooler, a middle
+>  schooler, a high school freshman, a high school sophomore, or a high school
+>  junior, I have never made so many mistakes.
```
Comment (zachlatta) [inline _posts/2016-04-28-hack-camp-is-back.md]: I don't think you intended to have the extra space in front of this blockquote.

File: _posts/2016-04-28-hack-camp-is-back.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Happy hacking!
+
+\- Max
+
+[hackcamp logo]: /assets/hackcamp_logo.svg "Thanks for making the logo Jessica!"
+[hackcamp group photo]: /assets/hackcamp_group_photo.jpg "Tom Preston-Werner showed up"
+[hackcamp]: /assets/hackcamp.jpg "Fantastic group picture with our third session"
+[hackcamp brainstorm]: /assets/hackcamp_brainstorming.jpg "Brainstorming with post-its"
+[hackcamp energizer]: /assets/hackcamp_energizer.jpg "I think this was our best impression of a cat"
```
Comment (zachlatta) [inline _posts/2016-04-28-hack-camp-is-back.md]: Use underscores in these types of links. There should also be space between "hack" and "camp".

- `[hack_camp_logo]
- `[hack_camp_group_photo]`
- etc

Comment (zachlatta) [inline _posts/2016-04-28-hack-camp-is-back.md]: (files should also be renamed to follow this format)

**Comment by zachlatta** [comment]
Content is mostly good, left a few comments. Final thing: photos should be compressed to be significantly smaller than their current size. Compare the size of https://github.com/hackclub/blog/blob/hack-camp-2016-post/assets/hackcamp_group_photo.jpg with https://github.com/hackclub/blog/blob/hack-camp-2016-post/assets/hackcamp_brainstorming.jpg.

**Comment by MaxWofford** [comment]
@zachlatta Addressed!

**Pull request merged**